### PR TITLE
Create menu header and tag

### PR DESCRIPTION
**Pull request by CBinh710**
https://github.com/users/CBinh710/projects/2/views/1?pane=issue&itemId=72043922
image

### CODE REVIEW

File: practice-one/src/components/menu.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  <h2 class="menu-heading"><span class="primary-color">Menu</span> That <span class="accent-color">Always</span> Make You Fall In <span class="primary-color">Love</span></h2>
+  <div class="menu-tag">
+    <div class="menu-change-button">Ramen</div>
+    <div class="menu-change-button">Breakfast</div>
+    <div class="menu-change-button">Lunch</div>
+    <div class="menu-change-button">Dinner</div>
+    <div class="menu-change-button">Mexican</div>
+    <div class="menu-change-button">Italian</div>
+    <div class="menu-change-button">Desserts</div>
+    <div class="menu-change-button">Drinks</div>
```
Comment (hungdo-agilityio) [inline practice-one/src/components/menu.html]: Don't repeat yourself, create component for this

File: practice-one/src/components/menu.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<section class="menu-container">
+  <h2 class="menu-heading"><span class="primary-color">Menu</span> That <span class="accent-color">Always</span> Make You Fall In <span class="primary-color">Love</span></h2>
```
Comment (hungdo-agilityio) [inline practice-one/src/components/menu.html]: Separate into multiple line

File: practice-one/src/styles/components/menu.css [hunk 3]
```diff
@@ -0,0 +1,26 @@
+.menu-heading {
+  font-size: 48px;
+  font-weight: 700;
+  line-height: 67px;
```
Comment (hungdo-agilityio) [inline practice-one/src/styles/components/menu.css]: variables?

File: practice-one/src/styles/components/menu.css [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  text-align: center;
+  padding: 0px 300px;
```
Comment (hungdo-agilityio) [inline practice-one/src/styles/components/menu.css]: `padding-inline: 300px`

File: practice-one/src/components/menu.html (hunk 1 above)
Comment (hungdo-agilityio) [reply practice-one/src/components/menu.html]: and also check the html tag

Comment (CBinh710) [reply practice-one/src/components/menu.html]: Thanks , i fixed it

File: practice-one/src/components/menu.html (hunk 2 above)
Comment (CBinh710) [reply practice-one/src/components/menu.html]: I fixed, thanks

File: practice-one/src/styles/components/menu.css (hunk 3 above)
Comment (CBinh710) [reply practice-one/src/styles/components/menu.css]: I added it

File: practice-one/src/styles/components/menu.css (hunk 4 above)
Comment (CBinh710) [reply practice-one/src/styles/components/menu.css]: Thanks for your comment

File: practice-one/src/components/menu.html [hunk 5]
```diff
@@ -0,0 +1,18 @@
+<section class="menu-container">
+  <h2 class="menu-heading">
+    <span class="primary-color">Menu</span>
+     That 
+    <span class="accent-color">Always</span>
+     Make You Fall In 
+    <span class="primary-color">Love</span></h2>
```
Comment (hungdo-agilityio) [inline practice-one/src/components/menu.html]: close tag `h2` should be on another line

File: practice-one/src/components/menu.html [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+  <div class="menu-tag">
+   <include src="./components/menu-button.html" locals='{"foodname": "Ramen"}'></include>
+   <include src="./components/menu-button.html" locals='{"foodname": "Breakfast"}'></include>
+   <include src="./components/menu-button.html" locals='{"foodname": "Lunch"}'></include>
+   <include src="./components/menu-button.html" locals='{"foodname": "Dinner"}'></include>
+   <include src="./components/menu-button.html" locals='{"foodname": "Mexican"}'></include>
+   <include src="./components/menu-button.html" locals='{"foodname": "Italian"}'></include>
+   <include src="./components/menu-button.html" locals='{"foodname": "Desserts"}'></include>
+   <include src="./components/menu-button.html" locals='{"foodname": "Drinks"}'></include>
+  </div>
```
Comment (hungdo-agilityio) [inline practice-one/src/components/menu.html]: should this is a list?

File: practice-one/src/styles/components/menu.css [hunk 7]
```diff
@@ -0,0 +1,14 @@
+.menu-heading {
+  font-size: var(--font-size-sub-header);
+  font-weight: 700;
+  line-height: 1.4;
```
Comment (hungdo-agilityio) [inline practice-one/src/styles/components/menu.css]: Do we need variable for font weight and line height similar to font size?

File: practice-one/src/components/menu.html (hunk 5 above)
Comment (CBinh710) [reply practice-one/src/components/menu.html]: thanks for your comment

File: practice-one/src/styles/components/menu.css (hunk 7 above)
Comment (CBinh710) [reply practice-one/src/styles/components/menu.css]: Yes, that's can be reused many time. So i will make it

File: practice-one/src/components/menu.html (hunk 6 above)
Comment (CBinh710) [reply practice-one/src/components/menu.html]: Thanks for your comment

**Review by hungdo-agilityio — APPROVED** [review]

**Pull request merged**